Scatt.java

```diff
@@ -27,13 +27,13 @@
      */
     public static void main(String[] args)
     {
-        System.out.println("Please enter the folder name: ");
+        System.out.print("Please enter the folder path: ");
         String dirName = System.console().readLine();
```

Comment:
The assumption thus far has been that the jar would be run from the current working directory, thus they would only have to enter the folder name.  I'm okay with this though.